### ISSUE TITLE
Suggestion for Improvement for Agents - Add a pop-up warning that you may be selecting an agent that is not associated with that collection type

### ISSUE THREAD

**Issue by ewommack** [ArctosDB/arctos#7204]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Is your feature request related to a problem? Please describe.**
How do we help Arctos users select the right Agents during Data Entry when there is only one person with a given name in Arctos? If there is only one person with a given name, then the pop-up window does not appear that provides you with the options of seeing the background information on that Agent. If you enter the name, and Arctos just confirms the person instead of making you choose from a list, then you may not realize that you should actually be creating a new Agent for your collection, and instead you just selected the wrong Agent for your Data Entry.

**Describe what you're trying to accomplish** 
One way we spot these incorrect Agents after the fact is by seeing that people may have records in different types of collections from their area of specialization. So if someone who specialized in herps, suddenly is associated with an item in a Geology collection, that may mean that it was selected in correctly. 

Is there a way we could flag what types of collections Agents are associated with in the Data Entry process, so you can spot this type of flag before you create the association?

**Describe the solution you'd like**
Is there a way to create a pop-up that might tell a Data Entry person that they just selected an Agent that is not associated with the collection type they are performing Data Entry in?
For example - I am entering data in the UWYMV:Bird collection in Arctos, and I select an Agent that is only associated with Entomology collections. I get a little pop-up window from Arctos when I hit return for the Agent name that tells me - "this Agent is not currently associated with your collection type. Are you sure this is the Agent you need?"

**Describe alternatives you've considered**
Find another way to help people better distinguish Agents where you enter the full name in Data Entry, and you do not have to go through the selection process from a list of Agents. Perhaps better training for data entry. Making it so people always have to select from a list (though that might be annoying for people who know exactly who they are using).

**Additional context**
This was brought up in discussion at the Diversity and Inclusion Committee meeting on the 19th of Dec. 2023

]

**Comment by dustymc** [comment]
This directly conflicts with https://github.com/ArctosDB/arctos/issues/6694.

https://github.com/ArctosDB/arctos/issues/6813 is the place for big-picture discussions.

**Comment by ewommack** [comment]
> This directly conflicts with https://github.com/ArctosDB/arctos/issues/6694

Yes we did discuss that as well. Our hope would be that it would be just a pop-up that does not happen for frequent users who are using people who they have already performed data entry for. If an Agent already has entries that are present in your type of collection, you would not get a pop-up window. Only if you selected a name, and the person had never contributed to the type of collection you were working in.
